deeppavlov/download.py

```diff
@@ -140,6 +146,7 @@ def deep_download(config: Union[str, Path, dict]) -> None:
     downloads = get_configs_downloads(config)
 
     for url, dest_paths in downloads.items():
+        url += "?config=" + config.stem
```

Comment:
what if the url already has a `?` character?

Reply:
added a def to treat this case